### ISSUE TITLE
Do we need something like a Relation-provider?

### ISSUE THREAD

**Issue by xrotwang** [clld/concepticon-data#299]
Looking at parabank, it seem as if relations among concepts could become one of the more important objects of Concepticon. So we may want to batch these relations by provider to make them more distinctive and also attributable.

**Comment by xrotwang** [comment]
@SimonGreenhill Does this make sense?

**Comment by xrotwang** [comment]
So in this scenario, parabank would be overlaying Concepticon data with lexibank data and add analyses.

**Comment by LinguList** [comment]
Actually, outsourcing relations as some extra kind of metadata seems ueseful, but I'd like to keep our current ones for the comparison of concept lists, as we now finally have achieved that despite a total of 210 concept sets as union, we have 207 comparison concepts when comparing Swadesh 100 and Swadesh 200.

But given that wordnet also has relations, and CLICS has it's own set of relations, we might start thinking of how to add this information in meta-data other than just saying "come and get it if you want it".

**Comment by xrotwang** [comment]
Agree. Thinking a bit more, paradigms may be better modelled as special concept lists, right?

**Comment by LinguList** [comment]
yes, agree.

**Comment by SimonGreenhill** [comment]
Yeah, I guess the parabanks contain 1:1 or 1:n mapping of relationships of conceptica. There's already something like that in concepticon with the concept relations:

```
559	BROTHER (OF MAN)	narrower	2414	OLDER BROTHER (OF MAN)
1759	OLDER BROTHER	narrower	2414	OLDER BROTHER (OF MAN)
```

-- is it possible to extend that relationship mapping?

**Comment by LinguList** [comment]
yes, but the question is: do we want to do this in concepticon, or instead find ways to annotate it in addition to a specific concept list, such as the parabank concept list, or meta data. E.g., word net has hierarchies, also more complex, and clics has a complex network. It may be useful to store this meta-data in concepticon but maybe without overriding the "normal" and "simple" relations.

**Comment by xrotwang** [comment]
It looks like a contribution to concepticon may consist of a concept list **plus** a set of relations. So we may want to make relations attributable, too. This may also allow distinguishing the core relations. I also think that paradigms may special as concept lists in that they probably don't work too well in our gloss-to-concept mapping algorithms.

**Comment by LinguList** [comment]
yes. I'd advocate a simple triple-system:

| source | target | relationA | relationB | etc. |
| --- | --- | --- | --- | --- |
| hand | arm | ... | ... | ... |

relation type (number, qualitative) should then be given in the metadata.

This is a basic network, if undirected, this should also be mentioned in metadata.

**Comment by xrotwang** [comment]
[Quads](https://en.wikipedia.org/wiki/Named_graph) actually, if we add in the relation provider. So the Concepticon editors would be the providers of the "core" or "simple" relations, @LinguList mentioned.

**Comment by LinguList** [comment]
Ah, the list we can take as a first example for this structure would be the one by Wilkins-1996-75, which I added using an ad-hoc way of representation (one target column, in which the concepts to which the given concept links are displayed).
